### PR TITLE
Enable trajectory output and correct input parsing

### PR DESCRIPTION
**Pull request by SG-phimeca**

### CODE REVIEW

File: otfmi/fmi.py [hunk 1]
```diff
@@ -408,3 +408,66 @@ def set_dict_value(model, dict_value):
...
+
+    dict_trajectory_sample = dict()
+    for name_output in model.getFMUOutputDescription():
+        dict_trajectory_sample[name_output] = np.column_stack(
+            [dd[name_output] for dd in list_dict_trajectory])
+
+    if time_interpolate is None:
+        list_time, _ = zip(*list_output)
+    else:
+        list_time = [time_interpolate for _ in xrange(len(list_output))]
```
Comment (jschueller) [inline otfmi/fmi.py]: xrange is python2-specific

seems this can be simplified to `[time_interpolate] * len(list_output)`, is this right ?

**Comment by jschueller** [comment]
you documented arguments but not output values

Fixes http://trac.openturns.org/ticket/936 ?

File: otfmi/fmi.py (hunk 1 above)
Comment (jschueller) [inline otfmi/fmi.py]: xrange is python2-specific

seems this can be simplified to `list_time = [time_interpolate] * len(list_output)`, is this right ?

Comment (SG-phimeca) [reply otfmi/fmi.py]: Not exactly. time_interpolate may be mutable, so [time_interpolate] * len(list_output) might produce unexpected results somewhere else. I'll rewrite the list comprehension without xrange.

**Pull request closed without merging**